src/sentry/spans/consumers/process/flusher.py

```diff
@@ -26,6 +27,74 @@
 logger = logging.getLogger(__name__)
 
 
+class MultiProducerManager:
+    """
+    Manages multiple Kafka producers for load balancing across brokers/topics.
+
+    Configure multiple producers in settings.py:
```

Comment:
server.py ?

Reply:
from a user POV it's filenames like `settings_prod.py` or `~/.sentry/conf.py`, i guess i can use the latter here?